scripts/validate-front-matter/script.py

```diff
@@ -574,7 +601,11 @@ def validate(content, fpath, validation):
         if 'last_review_date' in fm:
             if type(fm['last_review_date']) is datetime.date:
                 diff = todays_date - fm['last_review_date']
-                if diff > datetime.timedelta(days=365):
+                expiration = 365
```

Comment:
Now we support a new metadata field to specify the expiration in days, requested by @lyind

Reply:
Thank you, LGTM!

We need to test this.